themes/atomic/README.md

```diff
@@ -159,11 +159,18 @@ Three environment variables can be defined to rearrange the segments order. The
 
 `___ATOMIC_BOTTOM="char"`
 
-## Follow me
+### Contact / Social Media
 
-I'm on the social media.
+*Get the latest News about Web Development, Open Source, Tooling, Server & Security*
```

Comment:
Sorry, but I don't feel comfortable with this. This open source project is not a platform for promoting and advertising your own social media profiles.